src/react/client/features/analyze-story/components/story-task-modal.tsx

```
@@ -0,0 +1,119 @@
+  const buttonText = getButtonText(taskStatus, taskType)
+  const showResults = streamingContent || taskStatus === 'loading'
+
+  return (
+    <div className="flex flex-col h-full max-h-full space-y-6">
+      <div className="flex items-center justify-between flex-shrink-0">
+        <h2 className="text-lg font-semibold">{title}</h2>
+        <button
+          onClick={handleClose}
+          className="text-gray-400 hover:text-gray-200 text-xl"
+          aria-label="Close"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)